target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -78,6 +78,7 @@ ramips_setup_interfaces()
 	dir-320-b1|\
 	dir-610-a1|\
 	dir-615-h1|\
+	gb-pc1|\
```

Comment:
I can not fix it without knowing to which switch ports the RJ45 jacks are connected.